### DISCUSSION TITLE
Line breaks

### DISCUSSION THREAD

**Discussion by weilies** [dolanmiu/docx#989] [General]
I am using multiple blank new Paragraphs to create multiple line breaks. Is there a better way to write it?

```
    const doc = new Document({
      sections: [
        {
          children: [
            new Paragraph({
              children: [
                new TextRun({
                  text: 'Dismissal Letter (PIP)',
                  color: '0000FF',
                  bold: true,
                })
              ],
              alignment: AlignmentType.RIGHT,
            }),
            new Paragraph('\n'),
            new Paragraph('\n'),
            new Paragraph('\n'),
            new Paragraph('\n'),
            new Paragraph({
              children: [
                new TextRun({
                  text: 'PRIVATE & CONFIDENTIAL ',
                  underline: {
                    type: UnderlineType.DOUBLE,
                    color: '990011'
                  }
                })
              ]
            })
          ]
        }
      ]
    });
```

**Comment by PatriciaFLJacob** [comment]
You can use the break property in the TextRun of your last Paragraph:

    const doc = new Document({
      sections: [
        {
          children: [
            new Paragraph({
              children: [
                new TextRun({
                  text: 'Dismissal Letter (PIP)',
                  color: '0000FF',
                  bold: true,
                })
              ],
              alignment: AlignmentType.RIGHT,
            }),
            new Paragraph({
              children: [
                new TextRun({
                  text: 'PRIVATE & CONFIDENTIAL ', break: 4,
                  underline: {
                    type: UnderlineType.DOUBLE,
                    color: '990011'
                  }
                })
              ]
            })
          ]
        }
      ]
    });

**Reply by weilies** [reply]
No diff....
